include/nighthawk/client/factories.h

```diff
@@ -72,8 +72,8 @@ class RequestSourceFactory {
 class TerminationPredicateFactory {
 public:
   virtual ~TerminationPredicateFactory() = default;
-  virtual TerminationPredicatePtr create(Envoy::TimeSource& time_source, Envoy::Stats::Scope& scope,
-                                         Envoy::MonotonicTime time) const PURE;
+  virtual TerminationPredicatePtr create(Envoy::TimeSource& time_source,
```

Comment:
(optional) If you don't mind, can we document this interface since we are touching this? Optional, since this is unrelated to this PR.

Reply:
Maybe I should just go over all the interfaces, and document everything that isn't in one go in a separate PR? I'd be happy to.

Reply:
Assuming you don't mind doing that, that would be well worth the time and would help us when reading the code base.

Reply:
Filed https://github.com/envoyproxy/nighthawk/issues/268